src/EnergyPlus/WaterToAirHeatPumpSimple.hh

```diff
@@ -281,31 +282,26 @@ struct WaterToAirHeatPumpSimpleData : BaseGlobalStruct
 
     Real64 const CelsiustoKelvin; // Conversion from Celsius to Kelvin
 
-    int NumWatertoAirHPs;   // The Number of Water to Air Heat Pumps found in the Input
-                            // INTEGER        :: WaterIndex = 0                   ! Water index
-                            // INTEGER        :: Count = 0
+    int NumWatertoAirHPs; // The Number of Water to Air Heat Pumps found in the Input
+                          // INTEGER        :: WaterIndex = 0                   ! Water index
+                          // INTEGER        :: Count = 0
+    int AirflowErrPointer;
     bool GetCoilsInputFlag; // Flag set to make sure you get input once
     Array1D_bool MySizeFlag;
     Array1D_bool SimpleHPTimeStepFlag; // determines whether the previous operating mode for the coil and it's partner has been initialized
 
     Real64 SourceSideMassFlowRate; // Source Side Mass flow rate [Kg/s]
     Real64 SourceSideInletTemp;    // Source Side Inlet Temperature [C]
     Real64 SourceSideInletEnth;    // Source Side Inlet Enthalpy [J/kg]
-    Real64 LoadSideMassFlowRate;   // Load Side Mass flow rate [Kg/s]
     Real64 LoadSideInletDBTemp;    // Load Side Inlet Dry Bulb Temp [C]
     Real64 LoadSideInletWBTemp;    // Load Side Inlet Wet Bulb Temp [C]
     Real64 LoadSideInletHumRat;    // Load Side Outlet Humidity ratio
     Real64 LoadSideInletEnth;      // Load Side Inlet Enthalpy [J/kg]
     Real64 LoadSideOutletDBTemp;   // Load Side Outlet Dry Bulb Temp [C]
     Real64 LoadSideOutletHumRat;   // Load Side Outlet Humidity ratio
-    Real64 LoadSideOutletEnth;     // Load Side Outlet Enthalpy [J/kg]
-    Real64 QSensible;              // Load side sensible heat transfer rate [W]
-    Real64 QLoadTotal;             // Load side total heat transfer rate [W]
     Real64 QLatRated;              // Latent Capacity [W] rated at entering air conditions [Tdb=26.7C Twb=19.4C]
     Real64 QLatActual;             // Actual Latent Capacity [W]
-    Real64 QSource;                // Source side heat transfer rate [W]
```

Comment:
@yzhou601 this last change I made was to remove these redundant vars in the state struct since they did not add anything. I tried to be as simple as possible with changes but there could be a lot more clean up, now or later. I am relying on the CI to show the same diffs as before, or fewer since I pulled develop, and this or another issue can take it from here. Thanks for the focus on what caused this diff and the corrective action.